src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -10169,23 +10169,23 @@ internal void SetAcceptDrops(bool accept)
 
                     if (accept)
                     {
-                        Debug.WriteLineIf(CompModSwitches.DragDrop.TraceInfo, "Registering as drop target: " + Handle.ToString());
+                        Debug.WriteLineIf(CompModSwitches.DragDrop.TraceInfo, "Registering as drop target: " + Handle);
```

Comment:
Consider using string interpolation
```suggestion
                        Debug.WriteLineIf(CompModSwitches.DragDrop.TraceInfo, $"Registering as drop target: {Handle}");
```